### DISCUSSION TITLE
KFX Format

### DISCUSSION THREAD

**Discussion by mconnors625** [apprenticeharper/DeDRM_tools#2346] [Q&A]
See attached.  After importing the .azw files for this book into Calibre from K4PC the format list includes KFX.  Is that supposed to be there and does it mean Calibre was unable to do what it is supposed to do with the De**M 10.0.9 and KFX Input 2.15.0?  None of my other books have KFX showing in their format lists.  Is there a simple way to tell if Cailbre was successful at **M removal? 

Thank you.

![Screenshot 2024-09-02 145657](https://github.com/user-attachments/assets/47616157-37ba-472d-ac20-1b29471e4133)

**Comment by ElleKayEm** [comment]
If it hadn't worked, it would be KFX-ZIP.

**Reply by mconnors625** [reply]
Ok.  I seem to recall that now.  Thank you!
